Sources/FeedbacksKit/Model/FeedbackFormData.swift

```diff
@@ -1,12 +1,24 @@
+import UIKit
```

Comment:
❓ I can't check that now but I remember I added plateforme support for MacOS in the `Package.swift`. I'm wondering then if this won't break because UIKit can't be imported on MacOS.
Can you have a look ? It's not critical, you can remove MacOS support if needed 😄

Reply:
Yes, it will. I hadn't noticed there was Mac support in the `Package.swift` there are however several existing compilation issues, including existing UIKit imports, use of iOS specific SwiftUI view modifiers which mean it would require a bit more work to get running on Mac. For that reason, I've removed support as per your suggestion.